test/builder/auto_mocks_test.dart

```diff
@@ -3032,82 +3038,86 @@ void main() {
     );
   });
 
-  test(
-      'throws when GenerateMocks is given a class with a getter with a '
-      'non-nullable class-declared type variable type', () async {
-    _expectBuilderThrows(
-      assets: {
-        ...annotationsAsset,
-        ...simpleTestAsset,
-        'foo|lib/foo.dart': dedent('''
-        abstract class Foo<T> {
-          T get f;
-        }
-        '''),
-      },
-      message: contains(
-          "The property accessor 'Foo.f' features a non-nullable unknown "
-          'return type, and cannot be stubbed'),
-    );
-  });
-
-  test(
-      'throws when GenerateMocks is given a class with a method with a '
-      'non-nullable class-declared type variable return type', () async {
-    _expectBuilderThrows(
-      assets: {
-        ...annotationsAsset,
-        ...simpleTestAsset,
-        'foo|lib/foo.dart': dedent('''
-        abstract class Foo<T> {
-          T m(int a);
-        }
-        '''),
-      },
-      message: contains(
-          "The method 'Foo.m' features a non-nullable unknown return type, and "
-          'cannot be stubbed'),
-    );
-  });
-
-  test(
-      'throws when GenerateMocks is given a class with a method with a '
-      'non-nullable method-declared type variable return type', () async {
-    _expectBuilderThrows(
-      assets: {
-        ...annotationsAsset,
-        ...simpleTestAsset,
-        'foo|lib/foo.dart': dedent('''
-        abstract class Foo {
-          T m<T>(int a);
-        }
-        '''),
-      },
-      message: contains(
-          "The method 'Foo.m' features a non-nullable unknown return type, and "
-          'cannot be stubbed'),
-    );
-  });
-
-  test(
-      'throws when GenerateMocks is given a class with a method with a '
-      'non-nullable method-declared bounded type variable return type',
-      () async {
-    _expectBuilderThrows(
-      assets: {
-        ...annotationsAsset,
-        ...simpleTestAsset,
-        'foo|lib/foo.dart': dedent('''
-        abstract class Foo {
-          T m<T extends num>(int a);
-        }
-        '''),
-      },
-      message: contains(
-          "The method 'Foo.m' features a non-nullable unknown return type, and "
-          'cannot be stubbed'),
-    );
-  });
+  // TODO: please review and verify I can delete this test
+  //I think this is not not expected any more since we support non-nullable
+  // return types now.
```

Comment:
Please turn the "check that it fails" test into "check that it works" and verify that a call to `defaultDummyValue` is generated.

Reply:
and same for other tests that used to fail, but work now.